deploy/podman/compose.yaml

```diff
@@ -45,9 +63,36 @@ services:
     networks:
       - flightctl-network
 
+  flightctl-auth:
+    container_name: flightctl-auth
+    image: quay.io/keycloak/keycloak
+    ports:
+      - "9080:9080"
+      - "9443:9443"
+    command:
+      - start-dev
+      - --http-port=9080
+      - --https-port=9443
+      - --import-realm
+    environment:
+      KEYCLOAK_ADMIN: admin
+      KEYCLOAK_ADMIN_PASSWORD: adminpass
+      DB_VENDOR: postgres
+      DB_ADDR: flightctl-db:5433
+      DB_DATABSE: keycloak
+      DB_USER: keycloak
+      DB_PASSWORD: keycloak
+    volumes:
+      - ./realm-flightctl.json:/opt/keycloak/data/import/realm.json
```

Comment:
Could we move the realm-flightctl.json into a "keycloak" subdir just like with the prometheus et al.?

Reply:
updated